convert2rhel/actions/pre_ponr_changes/subscription.py

```diff
@@ -135,3 +143,11 @@ def run(self):
                 error_id="MISSING_REGISTRATION_COMBINATION",
                 message="One or more combinations were missing for subscription-manager parameters: %s" % e,
             )
+
+        if non_available_repos:
+            self.add_message(
+                level="WARNING",
+                id="NON_AVAILABLE_REPOSITORIES_DETECTED",
+                message="%s repositories are not available - some packages"
+                " may not be replaced and thus not supported." % ", ".join(non_available_repos),
+            )
```

Comment:
I think that to be on the safe side, we should move this condition to be inside the try-except, below line 110.

Since the other except statements don't have any return statement, my fear is that after setting the result in one of the exceptions, we will fall to this if statement and the variable might not exist at that point.

Reply:
Ok I will move it